### PR TITLE
Add PageBanner and Progress Bar components

### PR DESCRIPTION
**Pull request by villanuevawill**

### CODE REVIEW

File: src/containers/Profile/ProfileDetails.js [hunk 1]
```diff
@@ -22,6 +24,46 @@ import { ipfsToHttp } from 'utils/helpers';
...
+    name: user.name,
+    organization: user.organization,
+    languages: user.languages,
+    skills: user.skills,
+    email: user.email,
+    elsewhere: userElsewhereValues > 0 ? userElsewhereValues : undefined
+  };
+
+  const keys = Object.keys(userMinRequiredInfo).length;
+  const nonEmptyValues = Object.values(userMinRequiredInfo).filter(
```
Comment (c-o-l-o-r) [inline src/containers/Profile/ProfileDetails.js]: here you'll want to import `pickBy` from lodash and do `pickBy((key, value) => !isEmptyProperty(key), userMin)`

File: src/containers/Profile/ProfileDetails.js [hunk 2]
```diff
@@ -22,6 +24,46 @@ import { ipfsToHttp } from 'utils/helpers';
...
+    value => !isEmptyProperty(value)
+  ).length;
+
+  console.log(keys, nonEmptyValues);
```
Comment (c-o-l-o-r) [inline src/containers/Profile/ProfileDetails.js]: remove console logs

File: src/containers/Profile/ProfileDetails.js [hunk 3]
```diff
@@ -32,11 +74,40 @@ const ProfileDetailsComponent = props => {
...
   } = props;
 
   let bodyClass;
 
+  const percentageComplete = getProfileCompletePercent(user);
+
+  const pageBanner = (
+    <React.Fragment>
+      <PageBanner>
+        <span style={{ width: '50px' }} />
```
Comment (c-o-l-o-r) [inline src/containers/Profile/ProfileDetails.js]: Again, we don't use the style prop. Only className, so please update

Comment (c-o-l-o-r) [inline src/containers/Profile/ProfileDetails.js]: a lot of these spans will need to go away or turn to divs once this is switched to flexbox

File: src/containers/Profile/ProfileDetails.js [hunk 4]
```diff
@@ -105,10 +176,15 @@ const ProfileDetailsComponent = props => {
     body = <Loader color="blue" size="medium" />;
   }
 
-  return <div className={`col-xs-12 fullHeight ${bodyClass}`}>{body}</div>;
+  return (
+    <div>
+      {showBanner && percentageComplete < 100 && pageBanner}
+      <div className={`col-xs-12 fullHeight ${bodyClass}`}>{body}</div>
```
Comment (c-o-l-o-r) [inline src/containers/Profile/ProfileDetails.js]: this righ here is should be how you do all the styling -- by passing in a class (usually from the styles.scss you import

File: src/containers/Profile/ProfileDetails.js [hunk 5]
```diff
@@ -121,7 +197,8 @@ const mapStateToProps = state => {
     userStats,
     switchValue: profileUI.switchValue,
     currentTab: profileUI.currentTab,
-    loadedUserInfo
+    loadedUserInfo,
+    onEditProfile: props.onEditProfile
```
Comment (c-o-l-o-r) [inline src/containers/Profile/ProfileDetails.js]: If it is already in the props, what is the purpose of having it here? No state is needed?

File: src/containers/Profile/index.js [hunk 6]
```diff
@@ -115,21 +115,6 @@ class ProfileComponent extends React.Component {
   componentDidMount() {
     const body = document.getElementsByClassName('page-body')[0];
     body.addEventListener('scroll', this.onScroll);
-
-    const {
```
Comment (c-o-l-o-r) [inline src/containers/Profile/index.js]: is there a reason why this is getting removed? maybe you haven't rebased against master?

File: src/containers/Profile/index.js [hunk 7]
```diff
@@ -248,9 +245,7 @@ const Profile = compose(
       resetFilter: bountiesActions.resetFilter,
       loadUserInfo: userInfoActions.loadUserInfo,
       setActiveTab: actions.setActiveTab,
-      setProfileAddress: actions.setProfileAddress,
-      setReviewsModalVisible: actions.setReviewsModalVisible,
```
Comment (c-o-l-o-r) [inline src/containers/Profile/index.js]: again, is there a reason for removing these?

File: src/stories/PageBanner.stories.js [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+          Edit Profile
+        </Button>
+        <span style={{ float: 'right' }}>
+          <Button onClick={action('Close clicked')} type="link">
+            x
+          </Button>
+        </span>
+      </span>
+    </PageBanner>
+    <br />
```
Comment (c-o-l-o-r) [inline src/stories/PageBanner.stories.js]: Spamming br tags is actually a really bad practice as it breaks very easily, isn't as dynamic, and bloats the codebase. It is so much better adjust the spacing using styling like margins and padding. Could you remove these and achieve the intended effect with styling?

File: src/stories/PageBanner.stories.js [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+    <br />
+    <br />
+    <Text type="H3">
+      Light Blue PageBanner With ProgressBar, Edit and Close Buttons
+    </Text>
+    <br />
+    <br />
+    <PageBanner background="lightblue">
+      <span style={{ width: '50px' }} />
```
Comment (c-o-l-o-r) [inline src/stories/PageBanner.stories.js]: no style tag

File: src/stories/ProgressBar.stories.js [hunk 10]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import ProgressBar from '../components/ProgressBar';
+import PageBanner from '../components/PageBanner';
+import { Text } from 'components';
+
+storiesOf('ProgressBar', module).add('with Edit Profile', () => (
+  <div>
+    <br />
```
Comment (c-o-l-o-r) [inline src/stories/ProgressBar.stories.js]: same thing here with the br tags

File: src/components/PageBanner/PageBanner.css [hunk 11]
```diff
@@ -0,0 +1,20 @@
+.pageBanner {
+  width: 100%;
+  height: 25px;
```
[the remark this answers is not in the document]
Comment (villanuevawill) [reply src/components/PageBanner/PageBanner.css]: @bkatsnel also - you should be importing variables from variables.scss :) probably don't even use em explicitly.

File: src/containers/Profile/ProfileDetails.js [hunk 12]
```diff
@@ -22,6 +24,46 @@ import { ipfsToHttp } from 'utils/helpers';
 import { profileUISelector } from './selectors';
 import { actions } from './reducer';
 
+const isEmptyProperty = property => {
```
[the remark this answers is not in the document]
Comment (villanuevawill) [reply src/containers/Profile/ProfileDetails.js]: also lodash has a utility function for this if we did need it.

File: src/containers/Profile/ProfileDetails.js [hunk 13]
```diff
@@ -32,11 +74,40 @@ const ProfileDetailsComponent = props => {
     toggleNetworkSwitch,
     currentTab,
     setActiveTab,
-    setReviewsModalVisible
+    setReviewsModalVisible,
+    onEdit,
+    onCloseProgressBar,
```
Comment (villanuevawill) [inline src/containers/Profile/ProfileDetails.js]: There should be an onClose prop on the banner component. This shouldn't live on the progress bar.

File: src/containers/Profile/ProfileDetails.js [hunk 14]
```diff
@@ -32,11 +74,40 @@ const ProfileDetailsComponent = props => {
...
+          heading="Profile Strength"
+          percentage={percentageComplete}
+        />
+        <span style={{ width: '20px' }} />
+        <span className="width-stretch">
+          <Button onClick={onEdit} type="link">
+            Edit Profile
+          </Button>
+          <span style={{ float: 'right' }}>
+            <Button onClick={onCloseProgressBar} type="link">
```
Comment (villanuevawill) [inline src/containers/Profile/ProfileDetails.js]: The X should be built into the pagebanner component. And it should be there if the onClose prop is passed through to it.

File: src/components/PageBanner/index.js [hunk 15]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './PageBanner.css';
```
Comment (villanuevawill) [inline src/components/PageBanner/index.js]: this component needs a visible prop. visible = true or false which causes the pagebanner to show

**Comment by bkatsnel** [comment]
Will,
I will update the component based on your suggestions
Thanks, Boris

    On Wednesday, October 10, 2018, 6:39:06 PM EDT, William Villanueva <notifications@github.com> wrote:  
 
 
@villanuevawill commented on this pull request.

In src/components/PageBanner/index.js:
> @@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './PageBanner.css';

this component needs a visible prop. visible = true or false which causes the pagebanner to show

**Comment by bkatsnel** [comment]
Thanks, Matt. Let me look into this.
Boris

    On Wednesday, October 10, 2018, 6:29:13 PM EDT, Matt Garnett <notifications@github.com> wrote:  
 
 
@c-o-l-o-r requested changes on this pull request.

Hi @bkatsnel I'm Matt, I am an engineer at Bounties Network! 😄 I left some feedback on your PR for you to address. The biggest point of feedback to give you is to try and follow the patterns that are already set in place. We're using scss in all of our components to style them, so this would also need to use scss. Instead of px use rem. Instead of building style objects in the components, just pass them in as className props and import them from the scss. Etc. If you need help, be sure to checkout some of the other components and feel free to ping me on Slack.

In src/components/PageBanner/PageBanner.css:
> @@ -0,0 +1,20 @@
+.pageBanner {
+  width: 100%;
+  height: 25px;

we typically try to use rem wherever we can because it is more portable, so from here on I'll just tag instances that should be rem instead of px by just commenting rem.

In src/components/PageBanner/PageBanner.css:
> +  margin: 0;
+  padding: 0;
+  border-width: 0;
+  border-style: solid;
+  border-radius: 0;
+  border-color: grey;
+  background-color: lightgray;
+  overflow: hidden;
+  display: flex;
+  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.075);
+  align-items: center;
+}
+
+.width-stretch {
+  width: stretch;
+  margin-right: 30px;

rem

In src/components/PageBanner/PageBanner.css:
> @@ -0,0 +1,20 @@
+.pageBanner {
+  width: 100%;
+  height: 25px;
+  margin: 0;
+  padding: 0;
+  border-width: 0;
+  border-style: solid;
+  border-radius: 0;
+  border-color: grey;
+  background-color: lightgray;
+  overflow: hidden;
+  display: flex;
+  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.075);

we have shadow variables in the variables.scss file, it is best to use those to keep the styles across the site consistent

In src/components/PageBanner/PageBanner.css:
> @@ -0,0 +1,20 @@
+.pageBanner {

We use scss in our project, could you convert your css files over? The key for those styles to get picked up and transpiled by babel is using the naming convention of name.module.scss. If you're having trouble, just follow the pattern for import scss files in any of our other components.

In src/components/PageBanner/index.js:
> @@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './PageBanner.css';

scss

In src/components/PageBanner/index.js:
> @@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './PageBanner.css';
+
+const PageBanner = props => {
+  const { height, width, background } = props;

We try to not specify css within our components, instead we rely on passing classes through using className. Again, checkout any of our other components to familiarize yourself with that pattern.

In src/components/PageBanner/index.js:
> @@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './PageBanner.css';
+
+const PageBanner = props => {
+  const { height, width, background } = props;
+
+  const customStyles = {
+    height: height,
+    width: width,
+    backgroundColor: background
+  };
+
+  return (
+    <div className="pageBanner" style={customStyles}>

so here instead of style it would be className

In src/components/PageBanner/index.js:
> +  };
+
+  return (
+    <div className="pageBanner" style={customStyles}>
+      {props.children}
+    </div>
+  );
+};
+
+PageBanner.propTypes = {
+  height: PropTypes.string,
+  width: PropTypes.string,
+  background: PropTypes.string
+};
+
+PageBanner.defaultProps = {

good use of propTypes--however you won't need default props once you switch these out for just passing in a class

In src/components/ProgressBar/ProgressBar.css:
> @@ -0,0 +1,38 @@
+.progress-bar-all {
+  display: inline-block;

We usually prefer to utilize flexbox, especially for more complex components such as this? Could you convert this over to using flexbox? That will clear up a lot of other comments I'm going to leave.

In src/components/ProgressBar/ProgressBar.css:
> @@ -0,0 +1,38 @@
+.progress-bar-all {
+  display: inline-block;
+  white-space: nowrap;
+  overflow: hidden;
+  min-width: 500px;

rem

In src/components/ProgressBar/ProgressBar.css:
> @@ -0,0 +1,38 @@
+.progress-bar-all {
+  display: inline-block;
+  white-space: nowrap;
+  overflow: hidden;
+  min-width: 500px;
+}
+
+.progress-bar {
+  position: relative;
+  height: 5px;

rem -- also we have variables for these smaller numbers in variables.scss

In src/components/ProgressBar/ProgressBar.css:
> @@ -0,0 +1,38 @@
+.progress-bar-all {
+  display: inline-block;
+  white-space: nowrap;
+  overflow: hidden;
+  min-width: 500px;
+}
+
+.progress-bar {
+  position: relative;
+  height: 5px;
+  width: 350px;

rem

In src/components/ProgressBar/ProgressBar.css:
> @@ -0,0 +1,38 @@
+.progress-bar-all {
+  display: inline-block;
+  white-space: nowrap;
+  overflow: hidden;
+  min-width: 500px;
+}
+
+.progress-bar {
+  position: relative;
+  height: 5px;
+  width: 350px;
+  border-radius: 50px;

rem

In src/components/ProgressBar/ProgressBar.css:
> @@ -0,0 +1,38 @@
+.progress-bar-all {
+  display: inline-block;
+  white-space: nowrap;
+  overflow: hidden;
+  min-width: 500px;
+}
+
+.progress-bar {
+  position: relative;
+  height: 5px;
+  width: 350px;
+  border-radius: 50px;
+  border: 1px solid #333;

we have variables for borders

In src/components/ProgressBar/ProgressBar.css:
> @@ -0,0 +1,38 @@
+.progress-bar-all {
+  display: inline-block;
+  white-space: nowrap;
+  overflow: hidden;
+  min-width: 500px;
+}
+
+.progress-bar {
+  position: relative;
+  height: 5px;
+  width: 350px;
+  border-radius: 50px;
+  border: 1px solid #333;
+  display: inline-block;

flex

In src/components/ProgressBar/ProgressBar.css:
> @@ -0,0 +1,38 @@
+.progress-bar-all {
+  display: inline-block;
+  white-space: nowrap;
+  overflow: hidden;
+  min-width: 500px;
+}
+
+.progress-bar {
+  position: relative;
+  height: 5px;
+  width: 350px;
+  border-radius: 50px;
+  border: 1px solid #333;
+  display: inline-block;
+  color: white;

also, please use our variables for colors

In src/components/ProgressBar/ProgressBar.css:
> +  min-width: 500px;
+}
+
+.progress-bar {
+  position: relative;
+  height: 5px;
+  width: 350px;
+  border-radius: 50px;
+  border: 1px solid #333;
+  display: inline-block;
+  color: white;
+}
+
+.progress-bar-filler {
+  position: absolute;
+  background: #1da598;

variable

In src/components/ProgressBar/ProgressBar.css:
> +  display: inline-block;
+  color: white;
+}
+
+.progress-bar-filler {
+  position: absolute;
+  background: #1da598;
+  border-radius: inherit;
+  height: inherit;
+  transition: width 0.2s ease-in;
+  display: inline-block;
+}
+
+.progress-bar-hdr {
+  font-size: small;
+  margin-left: 5px;

rem

In src/components/ProgressBar/ProgressBar.css:
> +  color: white;
+}
+
+.progress-bar-filler {
+  position: absolute;
+  background: #1da598;
+  border-radius: inherit;
+  height: inherit;
+  transition: width 0.2s ease-in;
+  display: inline-block;
+}
+
+.progress-bar-hdr {
+  font-size: small;
+  margin-left: 5px;
+  margin-right: 10px;

rem

In src/components/ProgressBar/ProgressBar.css:
> +  border-radius: inherit;
+  height: inherit;
+  transition: width 0.2s ease-in;
+  display: inline-block;
+}
+
+.progress-bar-hdr {
+  font-size: small;
+  margin-left: 5px;
+  margin-right: 10px;
+}
+
+.progress-bar-pct {
+  font-size: small;
+  width: fit-content;
+  margin-left: 10px;

rem

In src/components/ProgressBar/ProgressBar.css:
> +  height: inherit;
+  transition: width 0.2s ease-in;
+  display: inline-block;
+}
+
+.progress-bar-hdr {
+  font-size: small;
+  margin-left: 5px;
+  margin-right: 10px;
+}
+
+.progress-bar-pct {
+  font-size: small;
+  width: fit-content;
+  margin-left: 10px;
+  margin-right: 10px;

rem

In src/components/ProgressBar/index.js:
> @@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './ProgressBar.css';
+
+const ProgressBar = props => {
+  const { heading, percentage, showPct, width } = props;

Can you change this to showPercent? Using the full words makes the code much more readable and maintainable.

In src/components/ProgressBar/index.js:
> +  const { heading, percentage, showPct, width } = props;
+
+  return (
+    <div className="progress-bar-all">
+      {heading && <span className="progress-bar-hdr">{heading}</span>}
+      <div className="progress-bar" style={{ width: width }}>
+        <Filler percentage={percentage} />
+      </div>
+      {showPct && <span className="progress-bar-pct">{percentage}%</span>}
+    </div>
+  );
+};
+
+const Filler = ({ percentage }) => {
+  return (
+    <div className="progress-bar-filler" style={{ width: `${percentage}%` }} />

We don't use the style prop, we always pass our styles in via a class to the className prop. Can you change please?

In src/components/ProgressBar/index.js:
> +  return (
+    <div className="progress-bar-filler" style={{ width: `${percentage}%` }} />
+  );
+};
+
+ProgressBar.propTypes = {
+  heading: PropTypes.string,
+  percentage: PropTypes.number.isRequired,
+  showPct: PropTypes.bool,
+  width: PropTypes.string
+};
+
+ProgressBar.defaultProps = {
+  showPct: true,
+  width: '350px'
+};

rem

In src/containers/Profile/ProfileDetails.js:
> @@ -22,6 +24,46 @@ import { ipfsToHttp } from 'utils/helpers';
 import { profileUISelector } from './selectors';
 import { actions } from './reducer';
 
+const isEmptyProperty = property => {

This seems pretty hacky, are all these checks really necessary? Wouldn't !(property && property.length) work?

In src/containers/Profile/ProfileDetails.js:
> +    property === undefined ||
+    property === null ||
+    (typeof property === 'object' && Object.keys(property).length === 0) ||
+    (typeof property === 'string' && property.trim().length === 0)
+  );
+};
+
+const getProfileCompletePercent = user => {
+  const userElsewhereInfo = {
+    github: user.github,
+    linkedin: user.linkedin,
+    twitter: user.twitter,
+    website: user.website
+  };
+
+  const userElsewhereValues = Object.values(userElsewhereInfo).filter(

We try to not use the Object constructors functions and instead use lodash. Here you'd want to import size from lodash and do const userElsewhereValues = size(userElsewhereInfo);

In src/containers/Profile/ProfileDetails.js:
> +  };
+
+  const userElsewhereValues = Object.values(userElsewhereInfo).filter(
+    value => value
+  ).length;
+
+  const userMinRequiredInfo = {
+    name: user.name,
+    organization: user.organization,
+    languages: user.languages,
+    skills: user.skills,
+    email: user.email,
+    elsewhere: userElsewhereValues > 0 ? userElsewhereValues : undefined
+  };
+
+  const keys = Object.keys(userMinRequiredInfo).length;

same thing here using lodash

In src/containers/Profile/ProfileDetails.js:
> +
+  const userElsewhereValues = Object.values(userElsewhereInfo).filter(
+    value => value
+  ).length;
+
+  const userMinRequiredInfo = {
+    name: user.name,
+    organization: user.organization,
+    languages: user.languages,
+    skills: user.skills,
+    email: user.email,
+    elsewhere: userElsewhereValues > 0 ? userElsewhereValues : undefined
+  };
+
+  const keys = Object.keys(userMinRequiredInfo).length;
+  const nonEmptyValues = Object.values(userMinRequiredInfo).filter(

here you'll want to import pickBy from lodash and do pickBy((key, value) => !isEmptyProperty(key), userMin)

In src/containers/Profile/ProfileDetails.js:
> +
+  const userMinRequiredInfo = {
+    name: user.name,
+    organization: user.organization,
+    languages: user.languages,
+    skills: user.skills,
+    email: user.email,
+    elsewhere: userElsewhereValues > 0 ? userElsewhereValues : undefined
+  };
+
+  const keys = Object.keys(userMinRequiredInfo).length;
+  const nonEmptyValues = Object.values(userMinRequiredInfo).filter(
+    value => !isEmptyProperty(value)
+  ).length;
+
+  console.log(keys, nonEmptyValues);

remove console logs

In src/containers/Profile/ProfileDetails.js:
>    } = props;
 
   let bodyClass;
 
+  const percentageComplete = getProfileCompletePercent(user);
+
+  const pageBanner = (
+    <React.Fragment>
+      <PageBanner>
+        <span style={{ width: '50px' }} />

Again, we don't use the style prop. Only className, so please update

In src/containers/Profile/ProfileDetails.js:
>    } = props;
 
   let bodyClass;
 
+  const percentageComplete = getProfileCompletePercent(user);
+
+  const pageBanner = (
+    <React.Fragment>
+      <PageBanner>
+        <span style={{ width: '50px' }} />

a lot of these spans will need to go away or turn to divs once this is switched to flexbox

In src/containers/Profile/ProfileDetails.js:
> @@ -105,10 +176,15 @@ const ProfileDetailsComponent = props => {
     body = <Loader color="blue" size="medium" />;
   }
 
-  return <div className={`col-xs-12 fullHeight ${bodyClass}`}>{body}</div>;
+  return (
+    <div>
+      {showBanner && percentageComplete < 100 && pageBanner}
+      <div className={`col-xs-12 fullHeight ${bodyClass}`}>{body}</div>

this righ here is should be how you do all the styling -- by passing in a class (usually from the styles.scss you import

In src/containers/Profile/ProfileDetails.js:
> @@ -121,7 +197,8 @@ const mapStateToProps = state => {
     userStats,
     switchValue: profileUI.switchValue,
     currentTab: profileUI.currentTab,
-    loadedUserInfo
+    loadedUserInfo,
+    onEditProfile: props.onEditProfile

If it is already in the props, what is the purpose of having it here? No state is needed?

In src/containers/Profile/index.js:
> @@ -115,21 +115,6 @@ class ProfileComponent extends React.Component {
   componentDidMount() {
     const body = document.getElementsByClassName('page-body')[0];
     body.addEventListener('scroll', this.onScroll);
-
-    const {

is there a reason why this is getting removed? maybe you haven't rebased against master?

In src/containers/Profile/index.js:
> @@ -248,9 +245,7 @@ const Profile = compose(
       resetFilter: bountiesActions.resetFilter,
       loadUserInfo: userInfoActions.loadUserInfo,
       setActiveTab: actions.setActiveTab,
-      setProfileAddress: actions.setProfileAddress,
-      setReviewsModalVisible: actions.setReviewsModalVisible,

again, is there a reason for removing these?

In src/stories/PageBanner.stories.js:
> +    <PageBanner>
+      <span style={{ width: '50px' }} />
+      <ProgressBar heading="Profile Strength" percentage={75} />
+      <span style={{ width: '20px' }} />
+      <span className="width-stretch">
+        <Button onClick={action('Edit clicked')} type="link">
+          Edit Profile
+        </Button>
+        <span style={{ float: 'right' }}>
+          <Button onClick={action('Close clicked')} type="link">
+            x
+          </Button>
+        </span>
+      </span>
+    </PageBanner>
+    <br />

Spamming br tags is actually a really bad practice as it breaks very easily, isn't as dynamic, and bloats the codebase. It is so much better adjust the spacing using styling like margins and padding. Could you remove these and achieve the intended effect with styling?

In src/stories/PageBanner.stories.js:
> +          <Button onClick={action('Close clicked')} type="link">
+            x
+          </Button>
+        </span>
+      </span>
+    </PageBanner>
+    <br />
+    <br />
+    <br />
+    <Text type="H3">
+      Light Blue PageBanner With ProgressBar, Edit and Close Buttons
+    </Text>
+    <br />
+    <br />
+    <PageBanner background="lightblue">
+      <span style={{ width: '50px' }} />

no style tag

In src/stories/ProgressBar.stories.js:
> @@ -0,0 +1,40 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import ProgressBar from '../components/ProgressBar';
+import PageBanner from '../components/PageBanner';
+import { Text } from 'components';
+
+storiesOf('ProgressBar', module).add('with Edit Profile', () => (
+  <div>
+    <br />

same thing here with the br tags

**Pull request closed without merging**